docs/acp/user/developing-an-app/developing-an-app.md

```diff
@@ -28,130 +38,201 @@ You need to have a clear understanding of what your app API will provide to othe
 Before you start with the development, you should design your API schema files. Depending on your requirements, you can have an OpenAPI or an [Async API](#async-api) schema file. In this step, you define the [Sync API](#sync-api) endpoints your app will provide to others, the messages you will emit or consume, and the data you expect to work with.
 
 {% info_block infoBox "Info" %}
-
 For more information about Async API schema design, see [Designing your APIs with Async API](https://www.asyncapi.com/blog/designing_your_apis_with_asyncapi_part_1). 
 
 For more information about OpenAPI schema design, see [Best practices in API design](https://swagger.io/resources/articles/best-practices-in-api-design/).
-
 {% endinfo_block %}
 
 You can use the following tools to design your APIs:
 - [Async API Studio](https://studio.asyncapi.com/)
 - [Swagger Editor](https://editor.swagger.io/)
 
-You can also use wizards provided by Spryker, which will be used by the [Spryker SDK workflow](/docs/sdk/dev/initialize-and-run-workflows.html).
+## Create an app
+
+Creating an app is straightforward. Execute the following commands:
+```bash
+mkdir my-app
+cd my-app
+git clone https://github.com/spryker-projects/mini-framework.git . --depth 1 && rm -rf .git && git init
+git add --all
+git commit -m "first commit"
+```
 
-## Build an app
+After these steps, you will have a new local repository that needs to be linked with your remote one.
 
-You can build a new app with the help of the [Spryker SDK](/docs/sdk/dev/spryker-sdk.html). After you have [installed the Spryker SDK](/docs/sdk/dev/spryker-sdk.html#installation), you can run the following commands to start building the app.
+If not done yet, create a new remote repository by opening your [Github account](https://github.com/newConnect). After you created a new repository, GitHub shows you instructions on how to continue. You should follow the list below as you don’t need some of the first steps proposed by GitHub as we’ve already initialized git and we already have a `README.md` from the cloned repository.
 
-1. Ensure that you are in the correct working directory:
+{% info_block infoBox "Info" %}
+This step can also be done later
+{% endinfo_block %}
 
 ```bash
-cd /www/my-app # The local project directory you created before
+git branch -M main
+git remote add origin git@github.com:<organization>/<repository>.git
+git push -u origin main
 ```
-2. Initialize the project:
+
+Now you have done the groundwork that enables you to develop an App. You created a new repository that contains the boilerplate code for almost any App you’d like to build.
+
+## Start local development environment
+
+The Mini-Framework already comes with a predefined Docker configuration. Change the `deploy.dev.yml` and replace `glue-backend.de.spryker.local` with `my-app.de.spryker.local` and then run the following command in the root of your new repository.
```

Comment:
This can potentially be part of the template, so that the developer does not have to rename anything.

Reply:
We have it in the final example 
see `Download the completed example from the Mini-Framework`